ui/app/components/job-page/parts/job-client-status-summary.js

```diff
@@ -8,10 +8,13 @@ import classic from 'ember-classic-decorator';
 export default class JobClientStatusSummary extends Component {
   job = null;
   jobClientStatus = null;
+  forceCollapsed = false;
   gotoClients() {}
 
-  @computed
+  @computed('forceCollapsed')
   get isExpanded() {
+    if (this.forceCollapsed) return false;
```

Comment:
Lets turn `forceCollapsed` into a reactive getter and then this no longer needs to be a computed property.

Reply:
Not sure if I follow. 

`forcedCollapsed` is an external property that is set by the caller to force the collapse state. `isExpanded` is the getter used to set the component state.